geopandas/base.py

```diff
@@ -699,8 +698,13 @@ def representative_point(self):
 
     @property
     def cascaded_union(self):
-        """Deprecated: Return the unary_union of all geometries"""
-        return cascaded_union(np.asarray(self.geometry.values))
+        """Deprecated: use `unary_union` instead"""
+        warn(
+            "The 'cascaded_union' attribute is deprecated, use 'unary_union' instead",
+            FutureWarning,
+            stacklevel=2,
+        )
+        return self.geometry.values.unary_union()
```

Comment:
In Shapely, `cascaded_union` was already performed by calling unary_union anyway for quite some time, so also using our `unary_union` implementation here, to avoid having to call `cascaded_uion` (which could otherwise a bit confusingly raise both the warning from geopandas as from shapely).